rules/msk_topic_config_comments.go

```diff
@@ -279,37 +280,43 @@ func (r *MSKTopicConfigCommentsRule) buildDurationComment(
 func buildCommentForMillis(timeMillis int, baseComment string) string {
 	timeUnits, unit := determineTimeUnits(timeMillis)
 
-	msg := fmt.Sprintf("# %s for %d %s", baseComment, timeUnits, unit)
+	timeUnitsStr := strconv.FormatFloat(timeUnits, 'f', -1, 64)
+	msg := fmt.Sprintf("# %s for %s %s", baseComment, timeUnitsStr, unit)
 	return msg
 }
 
-func determineTimeUnits(millis int) (int, string) {
-	// todo: this is not really perfect, as if the time is not exact in millis we'll output a partial number
-	timeInYears := millis / millisInOneYear
-	if timeInYears > 0 {
+/*	round to 1 digit precision  */
+func round(val float64) float64 {
```

Comment:
Added rounding, so that we support some slack in the values, as some teams consider a year 365 days, some other 365.25 and same for month.
With this rounding, I'm getting no diffs on the values currently in kafka-cluster-config